Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -169,9 +169,10 @@ - (void)setBridge:(RCTBridge *)bridge
 
 RCT_EXPORT_METHOD(removeAnimatedEventFromView:(nonnull NSNumber *)viewTag
                   eventName:(nonnull NSString *)eventName)
+            animatedNodeTag:(nonnull NSNumber *)animatedNodeTag
```

Comment:
You forgot to move the paren, so this extra arg isn't actually getting added to the native module signature :O

This diff is getting reverted, but I'll put it back up with the necessary fixes.

@mmmulani is working on a change that will make errors like this fail to compile rather than crashing at runtime in dev.